stdlib/2/tempfile.pyi

```diff
@@ -1,17 +1,4 @@
-# Stubs for tempfile
-# Ron Murawski <ron@horizonchess.com>
-
-# based on http: //docs.python.org/3.3/library/tempfile.html
-# Adapted for Python 2.7 by Michal Pokorny
-
-# TODO: Don't use basestring. Use Union[str, bytes] or AnyStr for arguments.
```

Comment:
None of these TODOs still applies: the file contains no mention of basestring and no Union return types.